line_item_manager/conf.d/line_item_template.yml

```diff
@@ -49,6 +49,23 @@ targeting:
       {% for placement in li.gam.placements|sort(attribute='id') %}
       - {{ placement.id }}
       {% endfor %}
+  {% if li.gam.geographies %}
+  geoTargeting:
+    {% if li.gam.geographies.include|length != 0 %}
+    includedLocations:
```

Comment:
I am not seeing this key supported here:
https://developers.google.com/ad-manager/api/reference/v202208/LineItemService.GeoTargeting

Reply:
My mistake, I only tested exclusions, should be fixed now